test/unit/message_test.rb

```diff
@@ -61,6 +62,33 @@ class MessageTest < ActiveSupport::TestCase
     end
   end
 
+  test "doesn't send email notifications to suspended buyers" do
+    Logic::RollingUpdates.expects(skipped?: true).at_least_once
+
+    sender = FactoryBot.create(:simple_provider)
+    approved_buyer = FactoryBot.create(:simple_buyer, provider_account: sender)
+    suspended_buyer = FactoryBot.create(:simple_buyer, provider_account: sender)
+    suspended_buyer.marked_for_suspension = true
+    suspended_buyer.suspend!
```

Comment:
```suggestion
    suspended_buyer = FactoryBot.create(:simple_buyer, provider_account: sender, state: 'suspended')
```

Not sure it's the same, but would be good if it works.

Reply:
Yes, this works: https://github.com/3scale/porta/pull/4078/commits/357bc07df4e27c4552a70e1f907e962d0c7cce76

Reply:
Do we support suspending buyer accounts? While they can have such state, perhaps test better be done with master->provider as the buyer instead of using this state for normal buyers what I think we do not use in real life. Just not to confuse future readers (us)

Reply:
If I'm not wrong, we suspend buyers when they don't pay:

https://github.com/3scale/porta/blob/01f16b96f0a89aea0d5b3278397feedeca6fdc6d/app/services/payment/multiple_failure_checker.rb#L11-L21